kornia/geometry/epipolar/numeric.py

```diff
@@ -37,7 +37,7 @@ def eye_like(n: int, input: torch.Tensor) -> torch.Tensor:
           The expected shape is :math:`(B, *)`.
 
     Returns:
-       The identity matrix with same size as input :math:`(*, N, N)`.
+       The identity matrix with same size as input :math:`(B, N, N)`.
```

Comment:
unrelated to this PR but do you think that we should move `eye_like` and `vec_like` under `kornia.utils ? and expose to top level. /cc @ducha-aiki @shijianjian

Reply:
I think those functions are already accessible from `kornia.eye_like` and `kornia.vec_like` respectively. But yeah I guess they can be moved to a more generic module like `konria.utils`